packages/adapter-squid/package.json

```diff
@@ -25,6 +24,9 @@
    "dependencies": {
       "@0xsquid/sdk": "^1.12.1",
       "@mate/sdk": "0.0.1",
+      "axios": "^1.6.0",
```

Comment:
axios is necessary because of the squid SDK? asking because it's still a relatively heavy package and seems to be only used for an error check in here so far

Reply:
That's right. If there are any underlying issues, they are only accessible via the axios error type. So instead of rewriting the same logic to check if the error is indeed an axios error it's more straightforward to use the built in function. On the package size, I don't think this is an issue as it allows the user to get detailed information on any potiential failures on the squid side.